internal/storage/sql/dialect.go

```diff
@@ -43,16 +43,18 @@ func (d *BaseDialect) CreateTableSQL(tableName string) string {
 			type VARCHAR(50) NOT NULL,
 			payload %s NOT NULL,
 			created_at %s NOT NULL,
+			forwarded_at %s,
 			status VARCHAR(20) NOT NULL,
 			error TEXT,
 			repository VARCHAR(255),
 			sender VARCHAR(255)
 		);
 		CREATE INDEX IF NOT EXISTS idx_created_at ON %s (created_at);
+		CREATE INDEX IF NOT EXISTS idx_forwarded_at ON %s (forwarded_at);
 		CREATE INDEX IF NOT EXISTS idx_type ON %s (type);
 		CREATE INDEX IF NOT EXISTS idx_status ON %s (status);
 		CREATE INDEX IF NOT EXISTS idx_repository ON %s (repository);
 		CREATE INDEX IF NOT EXISTS idx_sender ON %s (sender);
-	`, tableName, d.JSONType(), d.TimeType(),
-		tableName, tableName, tableName, tableName, tableName)
+	`, tableName, d.JSONType(), d.TimeType(), d.TimeType(),
+		tableName, tableName, tableName, tableName, tableName, tableName)
```

Comment:
Having to edit this across all the sql backends is making me want to wait for #12 before making this change. @tnm how close are we there?